### DISCUSSION TITLE
Input Data (Shape) Error while running sf.forecast()

### DISCUSSION THREAD

**Discussion by ABJ66** [Nixtla/statsforecast#773] [Q&A]
Hi all,

I am trying out various functionalities of statsforecast for univariate forecasting of multiple items. I'm running into this `ValueError: ValueError: could not broadcast input array from shape (32,1) into shape (54,1). `

Details for the setup/trial given as below:
1. Objective: forecast for multiple items which some of which have intermittent or have regular/non-intermittent history.
2. Data prepared in 3 column format: 'unique_id', 'ds', 'y'. Each item has exactly 54 time based measurements/values.
3. I've instantiated (a) StatsForecast class (as sf) with a bunch of models basis point # 1 and (b) asking for sf.forecast with h=6 and fitted = True, with input df basis point # 2.
4. It yields the `ValueError: could not broadcast input array from shape (32,1) into shape (54,1)`
5. Version 1.7.2 of statsforecast being used.
6. Earlier while using version 1.4.0, this error did not crop up, but the functionality of in-sample predictions was not implemented.

Would appreciate your inputs. Thank you in advance.

Regards.

**Comment by ABJ66** [comment]
Hi all, 

FYI: I am using CrostonClassic as one of the methods basis point # 1 above. Seems to be a bug in this method when data points are having value of zero. I found it here: https://github.com/Nixtla/statsforecast/issues/772. 

Pls let me know if this information is of use. 

I would like to view/test the predictions on some portion of in-sample data as well. There is a Qn. on StackOverflow: https://stackoverflow.com/questions/77214782/how-to-get-fitted-valuesinsample-forecast-for-model-where-fitted-true-is-not-i

Thank you for making this framework it is really helpful to quickly test out various algorithms. Look forward to your comments/suggestions.

Regards.

**Reply by jmoralez** [reply]
Hey. Yes, it's because of that issue, we're going to make a small release tomorrow that will include a fix. Sorry for the troubles.
